src/ci_providers/provider_azurepipelines.ts

```diff
@@ -66,7 +66,7 @@ function _getSHA(inputs: UploaderInputs): string {
     if (mergeCommitRegex.exec(mergeCommitMessage)) {
       const mergeCommit = mergeCommitMessage.split(' ')[1]
       info(`    Fixing merge commit SHA ${commit} -> ${mergeCommit}`)
-      commit = mergeCommit
+      commit = mergeCommit || ''
```

Comment:
```suggestion
      commit = mergeCommit ?? ''
```

The use of `??` circumvents some scenarios where `||` would consider the first operand to be false.[^1]

[^1]: [Nullish coalescing operator (??) - JavaScript | MDN]

[Nullish coalescing operator (??) - JavaScript | MDN]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator